common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/ScoreRequestTransformer.java

```diff
@@ -0,0 +1,62 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+import ai.h2o.mojos.deploy.common.rest.model.DataField;
+import ai.h2o.mojos.deploy.common.rest.model.Row;
+import ai.h2o.mojos.deploy.common.rest.model.ScoreRequest;
+import com.google.common.collect.ImmutableMap;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Transform scoring request rows,
+ * specifically, convert boolean literal string into 1, 0 respectively.
+ */
+public class ScoreRequestTransformer implements BiConsumer<ScoreRequest, List<DataField>> {
+
+  private static final Logger logger = LoggerFactory.getLogger(ScoreRequestTransformer.class);
+
+  @Override
+  public void accept(ScoreRequest scoreRequest, List<DataField> dataFields) {
+    Map<String, DataField> dataFieldMap =
+        dataFields.stream().collect(
+            ImmutableMap.toImmutableMap(DataField::getName, Function.identity()
+            )
+        );
+    scoreRequest.setRows(
+        transformRow(scoreRequest.getFields(), scoreRequest.getRows(), dataFieldMap)
+    );
+  }
+
+  List<Row> transformRow(List<String> fields, List<Row> rows, Map<String, DataField> dataFields) {
+    return rows.stream().map(row -> {
+      List<String> transformData = IntStream.range(0, row.size()).mapToObj(
+          fieldIdx -> {
+            String colName = fields.get(fieldIdx);
+            String origin = row.get(fieldIdx);
+            if (dataFields.containsKey(colName)) {
+              String sanitizeValue = Utils.sanitizeBoolean(
+                  origin, dataFields.get(colName).getDataType()
+              );
+              if (!sanitizeValue.equals(origin)) {
+                logger.info("Value '{}' parsed as '{}'", origin, sanitizeValue);
+              }
```

Comment:
`mojo2` runtime expose the same message on `INFO` level when transform boolean string literal into numeric value,  the only difference is `mojo2` can only handle lowercase or uppercase, because we doing something similar to `mojo2` which would has the equivalent of impact to the JAVA runtime, so i just keep the logging behavior consistent with existing convention.

Reply:
@jackjii79 if this is logging for every row that is not a good idea. The logs will get spammed.

Reply:
ok, it is indeed per row, changed to debug level https://github.com/h2oai/dai-deployment-templates/pull/327/commits/ba9ccdd75cd060f683f38a6a6642006be8459994